packages/transport/test/modules/sync/externalEventProcessors/PeerDeletedExternalEventProcessor.test.ts

```diff
@@ -35,8 +35,16 @@ describe("PeerDeletedExternalEventProcessor", function () {
 
     test("PeerDeletedExternalEventProcessor should mark peer as deleted", async function () {
         const eventProcessor = new PeerDeletedExternalEventProcessor(recipient.identityDeletionProcess.eventBus, recipient);
-        await eventProcessor.execute({ id: "anId", createdAt: "aDate", index: 1, syncErrorCount: 0, type: "PeerDeleted", payload: { relationshipId: relationshipId.toString() } });
+        await eventProcessor.execute({
+            id: "anId",
+            createdAt: "aDate",
+            index: 1,
+            syncErrorCount: 0,
+            type: "PeerDeleted",
+            payload: { relationshipId: relationshipId.toString(), deletionDate: CoreDate.local() }
+        });
         const relationship = await recipient.relationships.getRelationship(relationshipId);
         expect(relationship!.peerDeletionInfo!.deletionStatus).toBe(PeerDeletionStatus.Deleted);
+        expect(relationship!.peerDeletionInfo!.deletionDate.isSameOrBefore(CoreDate.local())).toBeTruthy();
```

Comment:
why do you test it like this? you can put the date in a variable and check the exact date. Same for other test.

Reply:
For this one yea in the other the date is create in the preFrom

Reply:
i meant the other processor. ANd you have already changed this.